### PR TITLE
RPR Test Script

### PR DESCRIPTION
**Pull request by spenceau**
### PURPOSE
Test Script to automate testing for RPR Blender Plug-in

### EFFECT OF CHANGE
cmd_render.py is a wrapper file that unzips the plugin and calls relevant files
add_script.py adds the subdir that extracted **rprblender** build to `bpy.context.preferences.filepaths.script_directories`
final_render.py to generate Final Render in background
remove_script.py to remove build subdir from `bpy.context.preferences.filepaths.script_directories` and copy over crash logs from LOCALAPPDATA in case of a failed render
compare_render.py to compare Final Render with Ground Truth using Pixel Wise MSE and SSIM
//viewport_render.py to generate Viewport Render (using final as specified) via opening Blender in foreground

run_render.cmd for Windows
run_render.sh for Ubuntu/Linux

this is stored within **cmd_tools/test_script**

### TECHNICAL STEPS
see README

### NOTES FOR REVIEWERS
cmd_tools/test_script/README.md specifies that this requires Python 3.11
viewport_render.py is deprecated as viewport render will probably not be used in the testing pipeline?

### CODE REVIEW

File: cmd_tools/test_script/viewport_render.py [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+    except ImportError as e:
+        print(f"Error importing rprblender: {e}")
+    except Exception as e:
+        print(f"Error registering rprblender: {e}")
+
+
+def main():
+    print("Starting viewport render script...")
+
+    # import bpy
```
Comment (feniksa) [inline cmd_tools/test_script/viewport_render.py:61]: I think, it is possible to drop this comment as argparse used below.

**Comment by odil24** [comment]
@spenceau What PR does?

**Review by feniksa — APPROVED** [review]

File: build.cmd [hunk 2]
```diff
@@ -64,14 +64,14 @@ if %vs_major%==15 or %vs_major%==16 (
...
-py -3.11 src\bindings\pyrpr\src\pyrprapi.py %castxml%
+python -m pip install cffi
+python -m pip install numpy
+python cmd_tools\create_sdk.py
+python src\bindings\pyrpr\src\pyrprapi.py %castxml%
 
 set bindingsOk=.\bindings-ok
 if exist %bindingsOk% (
-	py -3.11 build.py
+	python build.py
```
Comment (takahiroharada) [inline build.cmd:74]: Why? we shouldn't do these changes. Let's keep explicit python versioning.

**Review by takahiroharada — CHANGES_REQUESTED** [review]
Keep the changes minimum. SDK update shouldn't be in this PR. If needed, we need to do in another PR. Please do review all the changes and remove unnecessary files from this PR.

**Comment by spenceau** [comment]
Changed to merge fork into the RPRBLD-0-test-script

**Pull request closed without merging**